base/applications/cleanmgr/util.c

```diff
@@ -0,0 +1,1066 @@
+        StringCbCatW(TargetedDir, _countof(TargetedDir), L"\\Temp");
+        if (PathIsDirectoryW(TargetedDir))
+        {
+            sz.TempASize = DirSizeFunc(TargetedDir);
+            if (sz.TempASize == 1)
+            {
+                return FALSE;
+            }
+            StringCbCopyW(wcv.TempDir, _countof(wcv.TempDir), TargetedDir);
+            ZeroMemory(&TargetedDir, sizeof(TargetedDir));
+        }
+        SHGetFolderPathW(NULL, CSIDL_PROFILE, NULL, SHGFP_TYPE_CURRENT, TargetedDir);
+        StringCbCatW(TargetedDir, _countof(TargetedDir), L"\\Local Settings\\Temp");
+        if (PathIsDirectoryW(TargetedDir))
+        {
+            sz.TempBSize = DirSizeFunc(TargetedDir);
+            if (sz.TempBSize == 1)
+            {
+                return FALSE;
+            }
+            SendMessageW(HProgressBar, PBM_SETPOS, 25, 0);
+            SetDlgItemTextW(dv.hwndDlg, IDC_STATIC_INFO, L"Temporary Files");
+            StringCbCopyW(wcv.AltTempDir, _countof(wcv.AltTempDir), TargetedDir);
+            ZeroMemory(&TargetedDir, sizeof(TargetedDir));
+        }
+
+        Sleep(150);
+    }
+
+    if(FindRecycleBin(wcv.DriveLetter))
+    {
+        StringCchPrintfW(TargetedDir, _countof(TargetedDir), L"%s\\%s", wcv.DriveLetter, wcv.RecycleBin);
+        sz.RecycleBinSize = DirSizeFunc(TargetedDir);
+        if (sz.RecycleBinSize == 1)
+        {
+            return FALSE;
+        }
+        StringCbCopyW(wcv.RecycleBinDir, _countof(wcv.RecycleBinDir), TargetedDir);
+        ZeroMemory(&TargetedDir, sizeof(TargetedDir));
+        SendMessageW(HProgressBar, PBM_SETPOS, 50, 0);
+        SetDlgItemTextW(dv.hwndDlg, IDC_STATIC_INFO, L"Recycled Files");
+        Sleep(150);
+    }
+
+    
+    for(int i = 0; i < 10; i++)
+    {
+        StringCchPrintfW(TargetedDir, _countof(TargetedDir), L"%s\\FOUND.%.3i", wcv.DriveLetter, i);
+        if (PathIsDirectoryW(TargetedDir))
+        {
+            sz.ChkDskSize += DirSizeFunc(TargetedDir);
+            if (sz.ChkDskSize == 1)
+            {
+                return FALSE;
+            }
+            ZeroMemory(&TargetedDir, sizeof(TargetedDir));
+            SendMessageW(HProgressBar, PBM_SETPOS, 75, 0);
+            SetDlgItemTextW(dv.hwndDlg, IDC_STATIC_INFO, L"Old ChkDsk Files");
+        }
+        
+        else
+            break;
+    }
+    
+    /* HACK: Checking if there is a custom path for RAPPS (It will break with an update of RAPPS */
+    if(RegQueryValueExW(hRegKey, L"Settings", 0, NULL, (LPBYTE)&SettingsInfo, &dwSize) == ERROR_SUCCESS)
+        StringCbCopyW(TargetedDir, _countof(TargetedDir), SettingsInfo.szDownloadDir);
+
+    
+    /* Default path of RAPPS to store downloaded files */
+    else
+    {
+        DPRINT("RegQueryValueExW(): Failed to query a registry key!\n");
+        SHGetFolderPathW(NULL, CSIDL_PROFILE, NULL, SHGFP_TYPE_CURRENT, TargetedDir);
+        StringCbCatW(TargetedDir, _countof(TargetedDir), L"\\My Documents\\RAPPS Downloads");
```

Comment:
My Documents can be translated, so this is not correct.
At least use `CSIDL_PERSONAL` and append `RAPPS Downloads` to it.

Reply:
@learn-more Ah, I wasn't aware for that. Thanks.